tools/shared.py

```diff
@@ -46,6 +46,7 @@
                     level=logging.DEBUG if DEBUG else logging.INFO)
 colored_logger.enable()
 logger = logging.getLogger('shared')
+warnings_are_errors = False
```

Comment:
instead of adding this here, it would be on `Settings`, in which case JS can access it too, and we can make the warnings there errors as well (see the `warn, warnOnce` methods).

Reply:
Hmm, agreed.